### PR TITLE
Update Contributing.md

### PR DESCRIPTION
**Pull request by ajeetrocks100**
datastructure --> Data Structure (conventionally)
carzy extensive --> crazily (adverb)

### CODE REVIEW

**Comment by domfarolino** [comment]
Aw come on, if you're going to update the Contributing.md file, at least read it lol. Check out the commit message section and amend yours

**Comment by ajeetrocks100** [comment]
My bad!

**Comment by domfarolino** [comment]
@ajeetrocks100 If you want to open another PR that does the same thing go ahead, just mind the guidelines :p

**Comment by ajeetrocks100** [comment]
Okay, I'll try but guidelines are little confusing(for Docs).

File: CONTRIBUTING.MD [hunk 1]
```diff
@@ -30,7 +30,7 @@ If the formatting doesn't match the guide perfectly it's fine, just make sure yo
 ```
  - Open up a PR on GitHub and that's it!
 
-When adding an algorithm or datastructure to the repository, providing `test.in` and `test.out` stdin/stdout files demonstrating a procedure's correct output is a plus! These don't need to be crazy extensive, just enough to demonstrate some edge cases if necessary.
```
Comment (ajeetrocks100) [inline CONTRIBUTING.MD]: docs: Documentation changes only.  'x --> y' shows 'x' has been modified to 'y' 
datastructure --> Data Structure (conventionally)
crazy extensive --> crazily (adverb)

Comment (ajeetrocks100) [inline CONTRIBUTING.MD]: just wanted to show that it is a compound noun(" 'Data' 'Structure' ")
